<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DockWidget</class>
 <widget class="QDockWidget" name="DockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>341</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Options</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <widget class="QLabel" name="sideDistLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>27</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>201</width>
      <height>21</height>
     </size>
    </property>
    <property name="text">
     <string>Side distance (m) :</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sideDistSpinBox">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>27</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>999</number>
    </property>
    <property name="value">
     <number>15</number>
    </property>
   </widget>
   <widget class="QLabel" name="title2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>96</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Graphical visualisation</string>
    </property>
   </widget>
   <widget class="QLabel" name="title1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>4</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>251</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Variable for slope calculation</string>
    </property>
   </widget>
   <widget class="QLabel" name="toleratedASlopeLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>201</width>
      <height>41</height>
     </size>
    </property>
    <property name="text">
     <string>Tolerated slope (along) in percent :</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="toleratedASlopeSpinBox">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>120</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>300</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="swathDistLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>201</width>
      <height>41</height>
     </size>
    </property>
    <property name="text">
     <string>Swath distance for harvest (m) :</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="swathDistSpinBox">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>240</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>300</number>
    </property>
    <property name="value">
     <number>30</number>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>290</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QLabel" name="maxLengthLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>201</width>
      <height>41</height>
     </size>
    </property>
    <property name="text">
     <string>Maximum length tolerated (m) :</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="maxLengthSpinBox">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>200</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>300</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="toleratedCSlopeSpinBox">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>160</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>300</number>
    </property>
    <property name="value">
     <number>4</number>
    </property>
   </widget>
   <widget class="QLabel" name="toleratedCSlopeLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>201</width>
      <height>41</height>
     </size>
    </property>
    <property name="text">
     <string>Tolerated slope (cross) in percent :</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="maxLengthCheckBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Hold distance</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="swathDistCheckBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>171</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>Display buffer</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="interpolCheckBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>53</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>191</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>DEM Interpolate</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
